scripts/build.js

```diff
@@ -1,3 +1,9 @@
+#!/usr/bin/env node
+/**
+ * @fileoverview
+ * Builds the simple-icons-font based on the installed simple-icons dependency.
```

Comment:
Maybe this change here?

```suggestion
 * Builds the simple-icons-font package based on the installed simple-icons dependency.
```